content/en/docs/zero-code/python/_index.md

```diff
@@ -23,17 +23,29 @@ opentelemetry-bootstrap -a install
 The `opentelemetry-distro` package installs the API, SDK, and the
 `opentelemetry-bootstrap` and `opentelemetry-instrument` tools.
 
+{{% alert title="Note" color="info" %}}
+
+You must install a distro package to get auto instrumentation working. The
+`opentelemetry-distro` package contains the default distro to automatically
+configure some of the common options for users. For more information, see
+[OpenTelemetry distro](/docs/languages/python/distro/).
+
+{{% /alert %}}
+
```

Comment:
So, is your suggestion not to add the note? 
Although we have in all examples the command to install the opentelemetry-distro package, it may still not be so clear to all users that they need a distro package to get auto-instrumentation working. 

Example here: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2157

Reply:
Apologies, I missed that open discussion before merging the PR

Reply:
Let's leave it for now, but I think we need to do a major review of the python docs here, the "distro" page talks a lot about automatic instrumentation, so it should probably be moved over into the Zero-code section and be reworked

Reply:
@svrnm will create an issue to track this